vllm/model_executor/layers/fused_moe/rocm_aiter_fused_moe.py

```diff
@@ -333,7 +333,9 @@ def rocm_aiter_biased_group_topk(
     num_expert_group: int = 0,
     topk_group: int = 0,
     scoring_func: str = "sigmoid",
-    e_score_correction_bias: Optional[torch.Tensor] = None
+    e_score_correction_bias: Optional[torch.Tensor] = None,
+    num_share_fusion_replicas: int = 0,
+    routed_scaling_factor: Optional[float] = None,
```

Comment:
Seems like you should assert that these values are set to default here since they ignored